src/Lean/Meta/LazyDiscrTree.lean

```diff
@@ -167,7 +167,7 @@ private def shouldAddAsStar (fName : Name) (e : Expr) : MetaM Bool := do
 Eliminate loose bound variables via beta-reduction.
 
 This is primarily used to reduce pi-terms `∀(x : P), T` into
-non-dependend functions `P → T`.  The latter has a more specific
+nondependent functions `P → T`.  The latter has a more specific
```

Comment:
I think the current text is not incorrect, but:
```suggestion
non-dependent functions `P → T`.  The latter has a more specific
```
would be a better standard. Easier to parse visually.